pavement.py

```diff
@@ -552,7 +558,7 @@ def test_integration(options):
             sh('sleep 30')
             call_task('setup_data')
         sh(('python manage.py test %s'
-            ' --noinput --liveserver=localhost:8000' % name))
+            ' --noinput --liveserver=0.0.0.0:8000' % name))
```

Comment:
Will be used later by QGIS Server unittests. It needs to access from different network interface.